packages/mdc-textfield/README.md

```diff
@@ -355,6 +355,7 @@ Property | Value Type | Description
 `trailingIconAriaLabel` | `string` (write-only) | Proxies to the foundation's `setTrailingIconAriaLabel` method.
 `leadingIconContent` | `string` (write-only) | Proxies to the foundation's `setLeadingIconContent` method.
 `trailingIconContent` | `string` (write-only) | Proxies to the foundation's `setTrailingIconContent` method.
+`endAligned` | `boolean` | Aligns the input to the end if true, or start if false.
```

Comment:
Let's add the class name to the "Classes" section (line 260)

Reply:
Added!